pdf/model/pattern.go

```
@@ -81,21 +81,34 @@ func (this *PdfTilingPattern) IsColored() bool {
 	}
 }
 
-// Get the pattern cell's content stream.
+// GetContentStream returns the pattern cell's content stream
 func (this *PdfTilingPattern) GetContentStream() ([]byte, error) {
+	decoded, _, nil := this.GetContentStreamWithEncoder()
+	return decoded, nil
```

Should probably return the error if returned by GetContentStreamWithEncoder ?